Odysee/Controllers/Content/SearchViewController.swift

```diff
@@ -99,44 +99,25 @@ class SearchViewController: UIViewController,
             if winningClaims.count > 0 {
                 winningClaims
                     .sort(by: {
-                        Decimal(string: $1.meta?.effectiveAmount ?? "0")! >
-                            Decimal(string: $0.meta?.effectiveAmount ?? "0")!
+                        Decimal(string: $0.meta?.effectiveAmount ?? "0")! >
+                            Decimal(string: $1.meta?.effectiveAmount ?? "0")!
```

Comment:
No, this is incorrect. We want the obtain the claim with the highest effective amount, hence we use reverse sorting (descending order). The other changes look good. Please revert this comparison and then I can merge.

Reply:
Ah, I see the problem. Doing `$1 > $0` is equivalent to `$0 < $1`, which is ascending sorting.

So the fix would either be what I've done, or using `$1 < $0`.

(Just shorthand, the code would use the `.meta?.effectiveAmount ?? "0"` stuff)